scalalib/src/mill/scalalib/bsp/ScalaMetalsSupport.scala

```diff
@@ -3,23 +3,44 @@ package mill.scalalib.bsp
 import mill.api.experimental
 import mill.{Agg, T}
 import mill.define.Target
+import mill.scalalib.api.Util.isScala3
```

Comment:
This one is deprecated. Please use `mill.scalalib.api.ZincWorkerUtil` instead.

Maybe we should enable `-deprecation` in this module too, haha.

Reply:
👍🏼 fixed